packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
+    }
 
     return React.createElement(component, {
       ...fixture,
```

Comment:
That's not a valid fixture.

This is the type of fixture relevant to this PR: Mocking React children.
```jsx
const fixture = {
  foo: 'bar',
  children: [
     <p>I am wrapped inside a parent component</p>
  ],
};
```
You may be thinking about this: Mocking the state of child components (that have string *refs*).
```js
const fixture = {
  foo: 'bar',
  state: {
    children: {
      c1: {
        baz: 42,
      },
    },
  },
};
```
We can use a different medium like Slack if you have further questions.